### DISCUSSION TITLE
Offer black text on white background theme in community version for accessibility reasons

### DISCUSSION THREAD

**Discussion by donateur** [photoprism/photoprism#3366] [General]
Hi,
_Not sure if this is the right place to raise this, so please let me know if it should be moved!_

I suffer from a dry eye condition.  It makes white text on dark backgrounds difficult to read as I see ghosting off characters - especially when I'm tired at night.  The community version of Photoprism only has one theme.  Could you please offer a second reverse color theme (black text on a white background) in the Community version so people with vision problems can easily use the app?

I really appreciate Photoprism and believe that everyone deserves to be reimbursed fairly for their work but until I can convince my family to use it, I'm sorry I cannot justify subscribing.

Thank you!

**Comment by lastzero** [comment]
Our new release uses a dark theme by default because we thought it would be a welcome improvement for users of our free version. 

However, the Community Edition **also offers several non-dark themes**, for example Lavender, Onyx and Raspberry. Please let us know if these work for you! If not, we'll be happy to provide additional themes to improve your user experience.

**Reply by donateur** [reply]
Aha, I didn't realise that.  I clicked the first few themes and they all redirected me to the subscription option.

Thank you the Lavender theme is good for me!

**Reply by lastzero** [reply]
Excellent! We will change the order in the next release so the themes are easier to find.

**Reply by donateur** [reply]
Thanks!  Otherwise how about putting a Premium label or a :star: on the premium themes??

**Reply by lastzero** [reply]
Done, see:
- #3368

**Reply by lastzero** [reply]
Our demo has been updated, so you can test this:
- https://demo.photoprism.app/library/settings

**Reply by donateur** [reply]
Fantastic! That was very fast! Thanks!

**Comment by donateur** [comment]
Some themes with light backgrounds are still available: Lavender, Onyx and Raspberry.
